app/src/androidTest/java/org/mozilla/focus/activity/SettingsGeneralTest.kt

```diff
@@ -103,6 +122,24 @@ class SwitchLocaleTest {
         }
     }
 
+    @SmokeTest
+    @Test
+    fun changeDefaultBrowserTest() {
+        homeScreen {
+        }.openMainMenu {
+        }.openSettings {
+        }.openGeneralSettingsMenu {
+            clickSetDefaultBrowser()
+            if (Build.VERSION.SDK_INT <= 23) {
```

Comment:
Let's switch this to compare against `Build.VERSION_CODES.M` https://apilevels.com

Reply:
done